plugins/pencilblue/controllers/api/admin/site_settings/email/send_test.js

```diff
@@ -24,26 +24,34 @@ function SendTestEmail(){}
 //inheritance
 util.inherits(SendTestEmail, pb.FormController);
 
-SendTestEmail.prototype.onPostParamsRetrieved = function(post, cb) {
+SendTestEmail.prototype.render = function(cb) {
     var self = this;
 
-    var message = this.hasRequiredParams(post, this.getRequiredFields());
-    if(message) {
-        cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, message)});
-        return;
-    }
-
-    var options = {
-        to: post.email,
-        subject: 'Test email from PencilBlue',
-        layout: 'This is a successful test email from the PencilBlue system.',
-    };
-    pb.email.sendFromLayout(options, function(err, response) {
-        if(err) {
-            cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, JSON.stringify(err))});
+    this.getJSONPostParams(function(err, post) {
```

Comment:
In routes.js if you change the block that references this controller to:
```
{
            method: 'post',
            path: "/api/admin/site_settings/email/send_test",
            auth_required: true,
            access_level: pb.SecurityService.ACCESS_ADMINISTRATOR,
            controller: path.join(pb.config.docRoot, 'plugins', 'pencilblue', 'controllers', 'api', 'admin', 'site_settings', 'email', 'send_test.js'),
            content_type: 'application/json',
            request_body: ['application/json']
}
```

You won't need to call ```getJSONPostParams```.  Instead, you just need to call ```var post = this.body```.